antlr/src/main/antlr/FSH.g4

```diff
@@ -18,6 +18,8 @@ invariantMetadata:  description | expression | xpath | severity;
 
 valueSet:           KW_VALUESET SEQUENCE vsMetadata* vsComponent+;
 vsMetadata:         id | title | description;
+codeSystem:         KW_CODESYSTEM SEQUENCE csMetadata* code*;
```

Comment:
So this is the one where we *do* have that extra `definition` field for each concept, so the `code` parser rule isn't quite the right fit on its own.  We also need to add in that optional `"`-delimited `definition`.

Reply:
And in the importer we'll have to make sure that the `code` doesn't have a system before the `#` -- because in the CodeSystem context, all concepts should start w/ `#` (but that's hard to enforce in a clean way in the grammar since other contexts optionally allow the system).

Reply:
Thank you for reminding me about `definition`! I've added an extra optional `STRING` token to the new `concept` rule in order to capture it along with the `code`.